_data/banking.yml

```
@@ -750,46 +938,60 @@ websites:
       - sms
       - proprietary
     doc: https://www.ing.at/haeufige-fragen-faq/pin-zugangsdaten
+    regions:
+      - at
 
   - name: ING-DiBa Deutschland
     url: https://www.ing.de
     img: ing.png
     tfa:
       - sms
     doc: https://www.ing.de/kundenservice/sicherheit/psd2/
+    regions:
+      - de
 
   - name: Intesa Sanpaolo
     url: https://www.intesasanpaolo.com
     img: intesasanpaolo.png
     tfa:
       - hardware
     doc: https://www.intesasanpaolo.com/it/common/footer/sicurezza.html
+    regions:
+      - it
 
   - name: Jovia Financial
     url: https://www.joviafinancial.com/
     img: jovia.png
     tfa:
       - sms
       - phone
+    regions:
+      - us
 
   - name: KeyBank
     url: https://www.key.com
     twitter: KeyBank_Help
     img: keybank.png
+    regions:
+      - us
 
   - name: La Banque Postale
     url: https://www.labanquepostale.fr/
     twitter: LaBanquePostale
     facebook: labanquepostale
     img: labanquepostale.png
     lang: fr
+    regions:
+      - fr
 
   - name: Lake Michigan Credit Union
     url: https://www.lmcu.org/
     img: lakemichigancreditunion.png
     tfa:
       - proprietary
     doc: https://www.lmcu.org/personal/banking/services/mobile-authenticator/
+    regions:
+      - fr
```

```suggestion
      - us
```
Lake Michigan Credit Union is a credit union in the US. It's insured by NCUA.